src/components/layout/BaseLayout.tsx

```diff
@@ -29,6 +31,7 @@ const BaseLayout: FC<PropsWithChildren<Props>> = ({
           <Divider />
         </>
       ) : null}
+      {heroAd ? heroAd : null}
```

Comment:
with this being outside of the content flow, how do you solve it that the search box covers parts of the image but the skyscraper at only starts after the hero ad?

Reply:
This solves the content and skyscraper ad to be in alignment like it should and like we have it now. The Search box will then need to be moved manually with `**-**margin-top` inside content area, which will be done on listings-web level. Probably some play around with `position: relative/absolute` to achieve that or at least that's how my idea was to get around it